examples/components/Radar.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <v-wedge :config='config'></v-wedge>
```

Comment:
You can add `ref="wedge"` so you can access it later from `methods`

Reply:
I appreciate that you gave comments. I will update it